README.md

```diff
@@ -20,19 +20,19 @@ Features:
 * Leverages SpikeExtractor to support conversion from a number or proprietary formats.
 
 ## Installation
-To install **nwb-conversion-tools** directly in an existing environment:
+Fore easy of use we recommend to install **nwb-conversion-tools** latest stable release in an existing environment with the following command:
```

Comment:
There's 3 ways someone can install the package.

1 is stable pip release, easiest and most recommended method.
2 is local git clone and conda setup (no pip at all)
3 is local pip install +/- local git clone (since you can just do `pip install git+https://github.com/catalystneuro/nwb-conversion-tools.git@main` to get latest dev branch)

Reply:
Ok, so right now we are missing the third one.